jdbc/src/test/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaDataTest.java

```diff
@@ -292,17 +349,38 @@ private void testGetOfColInfo(
     }
   }
 
+  /**
+   * Validate resultSet MetaData returned from getSchemas.
+   *
+   * @param resultSet ResultSet need to be validated.
+   * @throws SQLException If an error occurs while retrieving the value.
+   */
+  private void testGetSchemasResult(ResultSet resultSet) throws SQLException {
+    final String[] string1 = {"", "testDB", null};
```

Comment:
Can we have a test that expects > 1 row?.  Let us say SHOW DATABASES returns 2 rows when there are 2 databases creates in the account in the given region.

Reply:
The test with two rows returned has been added.